easybuild/easyconfigs/l/libharu/libharu-2.3.0-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
+
+name = 'libharu'
+version = '2.3.0'
+
+homepage = 'http://libharu.org/'
```

Comment:
Please change this to be the Github page, which is actively maintained and uses https.